ember/app/components/context-menu/template.hbs

```diff
@@ -0,0 +1,25 @@
+<UkButton
+  @type="button"
+  @color="text"
+  @title={{t "global.openMenu"}}
+>
+  <span hidden>{{t "global.openMenu"}}</span>
+  <UkIcon @icon="more-vertical" />
+</UkButton>
+<div uk-drop>
+  <div class="uk-card uk-card-body uk-card-small uk-card-default">
```

Comment:
I think you want to use the `uk-dropdown` here - not a card in a drop

Reply:
Changed